### PR TITLE
Rename column resize handle

### PR DESCRIPTION
**Pull request by pradeepnschrodinger**
## Description
Rename FixedDataTableColumnResizeHandle to ColumnResizerLine

## Motivation and Context
See #408

## How Has This Been Tested?
Ran the example for Column resizing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: src/FixedDataTableColumn.js [hunk 1]
```diff
@@ -149,9 +149,9 @@ class FixedDataTableColumn extends React.Component {
    flexGrow: PropTypes.number,
 
    /**
-    * Whether the column can be resized with the
-    * FixedDataTableColumnResizeHandle. Please note that if a column
-    * has a flex grow, once you resize the column this will be set to 0.
+    * Whether the column can be resized with the ColumnResizerLine.
```
Comment (wcjordan) [inline src/FixedDataTableColumn.js]: Should this actually be resize handle in FixedDataTableCell?

Comment (pradeepnschrodinger) [reply src/FixedDataTableColumn.js]: @wcjordan, hmm, technically you're right. It's the resize handle that initiates the ColumnResizerLine.
Do you think I should just refer to "the resize handle" here? (There's no react class here)

Comment (wcjordan) [reply src/FixedDataTableColumn.js]: Ya, I would just say whether the column can be resized.

Comment (pradeepnschrodinger) [reply src/FixedDataTableColumn.js]: sure, will drop it.

**Pull request merged**